.gitignore

```diff
@@ -27,4 +27,7 @@ node_modules
 # Users Environment Variables
 .lock-wscript
 
-govuk_template.html
\ No newline at end of file
+govuk_template.html
+
+# IDE
+.idea
```

Comment:
Have this in a global gitignore on your machine rather than project specific. A project .gitignore file should only exclude files and directories that are a part of the package that should not be versioned (such as the node_modules directory) as well as files that are generated as artifacts of a build process

Reply:
I have removed this.